src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -60,6 +62,8 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.